Scripts/Data/Type/Transformation/Vector2ToAngle.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace Zinnia.Data.Type.Transformation
+{
+    using UnityEngine;
+    using UnityEngine.Events;
+    using System;
+    using Zinnia.Extension;
+
+    /// <summary>
+    /// Transforms a <see cref="Vector2"/> value to a <see cref="float"/> angle.
+    /// </summary>
+    /// <remarks>
+    /// The origin angle (i.e. zero) is set to the top by default to mimic a clock with 12 o'clock being the 0 angle.
+    /// </remarks>
+    /// <example>
+    /// Vector2[0f, 0f] -> unit.degrees -> origin.Vector2(0f, 1f) = 0f
+    /// Vector2[1f, 0f] -> unit.degrees -> origin.Vector2(0f, 1f) = 90f
+    /// Vector2[0f, -1f] -> unit.degrees -> origin.Vector2(0f, 1f) = 180f
+    /// </example>
+    public class Vector2ToAngle : Transformer<Vector2, float, Vector2ToAngle.UnityEvent>
+    {
+        /// <summary>
+        /// Defines the event with the transformed <see cref="float"/> value.
+        /// </summary>
+        [Serializable]
+        public class UnityEvent : UnityEvent<float>
+        {
+        }
+
+        public enum AngleUnit
+        {
+            /// <summary>
+            /// The angle in degrees.
+            /// </summary>
+            Degrees,
+            /// <summary>
+            /// The angle in radians.
```

Comment:
```suggestion
            /// Measurement in radians. A full rotation is 2 * PI radians.
```
This isn't the `angle` after all.